packages/dso-toolkit/src/components/renvooi/renvooi.stories-of.ts

```diff
@@ -0,0 +1,57 @@
+import { ComponentAnnotations, Renderer } from "@storybook/types";
+
+import { RenvooiArgs, renvooiArgs, renvooiArgTypes, renvooiArgsMapper } from "./renvooi.args.js";
+import { Renvooi } from "./renvooi.models.js";
+
+import { StoriesParameters, StoryObj } from "../../template-container.js";
+import { compiler } from "markdown-to-jsx";
+import { MetaOptions } from "../../storybook/meta-options.interface.js";
+
+type RenvooiStory = StoryObj<RenvooiArgs, Renderer>;
+
+interface RenvooiStories {
+  Default: RenvooiStory;
+}
+
+interface RenvooiStoriesParameters<Implementation, Templates, TemplateFnReturnType>
+  extends StoriesParameters<
+    Implementation,
+    Templates,
+    TemplateFnReturnType,
+    RenvooiTemplates<TemplateFnReturnType>
+  > {}
+
+interface RenvooiTemplates<TemplateFnReturnType> {
+  renvooiTemplate: (renvooiProperties: Renvooi) => TemplateFnReturnType;
+}
+
+export function renvooiMeta<TRenderer extends Renderer>({ readme }: MetaOptions = {}): ComponentAnnotations<
+  TRenderer,
+  RenvooiArgs
+> {
+  return {
+    argTypes: renvooiArgTypes,
+    args: renvooiArgs,
+    parameters: {
+      docs: readme
+        ? {
+            page: () => compiler(readme),
+          }
+        : {},
+    },
+  };
+}
+
+export function renvooiStories<Implementation, Templates, TemplateFnReturnType>({
+  storyTemplates,
+  templateContainer,
+}: RenvooiStoriesParameters<Implementation, Templates, TemplateFnReturnType>): RenvooiStories {
+  return {
+    Default: {
+      args: {},
```

Comment:
Overschrijf je hier nu de renvooiArgs uit de renvooiMeta??

Reply:
Geen idee eigenlijk. Dit is hoe we onze plop templates hebben ingericht. Ff kijken

Reply:
Interessant. De boilerplate `componentArgs` (in dit geval `renvooiArgs`) wordt bij initialisatie van een nieuw component nergens gebruikt.

Reply:
Zoals afgesproken: Boilerplate bijgewerkt, dit stond onterecht in de stories-of template.